topics/transcriptomics/tutorials/RNA_seq_analysis/tutorial.md

```diff
@@ -0,0 +1,547 @@
+---
+layout: tutorial_hands_on
+
+title: 'From reads to (pseudo) counts : guided step by step exploration'
+zenodo_link: https://doi.org/10.5281/zenodo.10652085
+questions:
+- How to go from FASTA mRNA sequences to the study of differential gene expression?
+- How to manage isoforms and polymorphism?
+objectives:
+- An overview of RNA-seq data analysis steps
+- Identify the (key issues/points) (critical steps/parameters)
+- Introduction to assembly and (pseudo)counting tools
+time_estimation: 8H
+key_points:
+- This course will NOT allow you to design an analysis pipeline set-up for your specific needs
+- This course WILL give you the basis information to understand and run a generic RNA-seq analysis, its key steps and problematics, and how to interact with bioinformaticians/bioanalysts that can analyze your RNA-seq datasets
+supporting_materials:
+  available_on_these_galaxies:
+    known_working:
+      - UseGalaxy.fr
+      - UseGalaxy.fr   
```

Comment:
This is not a known metadata value, it will be flagged by our linter once I hit approve. Supported servers is calculated automatically by the GTN, no action needed on your part. :)